[windows] fix doxygen comments on GUIMediaWindow.cpp

AlwinEsch opened the pull request:
<!--- Provide a general summary of your change in the Title above -->

## Description
<!--- Describe your change in detail -->
On GUIMediaWindow.cpp was on many places the `\brief` used
but not the line started with `\\\`, `\*!` or `\**`, this
change to use with doxygen.

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here -->

## How Has This Been Tested?
<!--- Please describe in detail how you tested your change -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc -->

## Screenshots (if appropriate):

## Types of change
<!--- What type of change does your code introduce? Put an `x` in all the boxes that apply like this: [X] -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply like this: [X] -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] My code follows the [Code guidelines](https://codedocs.xyz/xbmc/xbmc/code_guidelines.html) of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/xbmc/blob/master/CONTRIBUTING.md) document
- [ ] I have added tests to cover my change
- [x] All new and existing tests passed

AlwinEsch commented:
jenkins build this please

akva2 commented on xbmc/windows/GUIMediaWindow.cpp:
```diff
@@ -1417,9 +1445,11 @@ bool CGUIMediaWindow::OnPlayAndQueueMedia(const CFileItemPtr &item, std::string
-// on the fileitems of the window
+/*!
+ * \brief Synchronize the fileitems with the playlistplayer
+ * It recreated the playlist of the playlistplayer based
```
that wasn't very brief. i'd suggest using \details for everything but the first line..

AlwinEsch replied:
I can change to
```
/*!
 * \brief Update file list
 *
 * Synchronize the fileitems with the playlistplayer
 * It recreated the playlist of the playlistplayer based
 * on the fileitems of the window
 */
```
then is it also used as \details

akva2 replied:
(y)

akva2 replied:
also recreateS

AlwinEsch replied:
Why not the emoticons has brought my a good smyle and the urge to find a woman ;-)

akva2 replied:
lol yeah that's what we used to draw on the blackboard to annoy the teacher back in the day .D

AlwinEsch committed:
[windows] fix doxygen comments on GUIMediaWindow.cpp

On GUIMediaWindow.cpp was on many places the `\brief` used
but not the line started with `\\\`, `\*!` or `\**`, this
change to use with doxygen.
